src/lib/forms/ArrayField.js

```diff
@@ -31,6 +33,32 @@ export class ArrayField extends Component {
     return false;
   };
 
+  /**
+   * Returns the array of values to display, it checks for required options and adds empty rows with the required values prefilled
+   * @param {Array} values The array of values
+   * @param {String} fieldPath The path of the field
+   * @returns An array of values to display
+   */
+  getValues = (values, fieldPath) => {
+    const { requiredOptions, defaultNewValue, showEmptyValue } = this.props;
+    const existingValues = getIn(values, fieldPath, []);
```

Comment:
not a blocker:
```suggestion
    const existingValues = getIn(values, fieldPath, []);
    const displayValues = [];
```
this is proposed to avoid pushing later on to existingValues, since they come from the backend and are the "source of truth" in this block of code